Classes/Hooks/DataHandler.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/**
+ * @license GPLv3, http://www.gnu.org/copyleft/gpl.html
+ * @copyright Gilbertsoft (gilbertsoft.org), 2017
+ * @copyright Aimeos (aimeos.org), 2017-
+ * @package TYPO3
+ */
+
+namespace Aimeos\Aimeos\Hooks;

```

Comment:
Yes, but this directory seems to be the standard at t3 sysext, e.g. have a look at the core! What should I do?